databricks/koalas/frame.py

```diff
@@ -8331,6 +8331,92 @@ def quantile(self, q=0.5, axis=0, numeric_only=True, accuracy=10000):
 
         return DataFrame(internal) if not result_as_series else DataFrame(internal).T[key]
 
+    def query(self, expr, inplace=False):
+        """
+        Query the columns of a DataFrame with a boolean expression.
+
```

Comment:
@itholic, can you elaborate what works and what doesn't work based on https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#the-query-method ?

Also, actually sorry I think `DataFrame.apply` doesn't work as a workaround as it iterates each series.

I will make a API like [`map_partitions`](https://docs.dask.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.map_partitions) soon which can actually work around. I think you should include the workaround example as well.

Reply:
> I think you should include the workaround example as well.

Totally agree. we'd better this for all docstring of other APIs which can work around.

Reply:
@itholic, here's the PR https://github.com/databricks/koalas/pull/1276. This will be able to workaround (see the example in the docstring).

Reply:
@HyukjinKwon 

Thanks for the comment!! i just added :)